fsi/fsi.go

```diff
@@ -253,6 +259,48 @@ func removeLinkFile(dir string) error {
 	return os.Remove(dir)
 }
 
+func deleteLowValueFiles(path string, f os.FileInfo, err error) error {
+	if f != nil {
+		filesToBeRemoved := []string{
+			// generic files
+			"^\\..*\\.swp$", // Swap file for vim state
+
+			// macOS specific files
+			"^\\.DS_Store$", // Stores custom folder attributes
+			"^\\.AppleDouble$", // Stores additional file resources
+			"^\\.LSOverride$", // Contains the absolute path to the app to be used
+			"^Icon\\r$", // Custom Finder icon: http://superuser.com/questions/298785/icon-file-on-os-x-desktop
+			"^\\._.*", // Thumbnail
+			"^\\.Spotlight-V100(?:$|\\/)", // Directory that might appear on external disk
+			"\\.Trashes", // File that might appear on external disk
+			"^__MACOSX$", // Resource fork
+
+			// linux specific files
+			"~$", // Backup file
+
+			// Windows specific files
+			"^Thumbs\\.db$", // Image file cache
+			"^ehthumbs\\.db$", // Folder config file
+			"^Desktop\\.ini$", // Stores custom folder attributes
+			"@eaDir$", // Synology Diskstation "hidden" folder where the server stores thumbnails
+		}
+
+		pattern := strings.Join(filesToBeRemoved, "|")
+		matched, err := regexp.MatchString(pattern, filepath.Base(f.Name()))
+		if err != nil {
+			return err
+		}
+		if matched {
+			os.Remove(path)
+		}
+	}
+	return nil
```

Comment:
My only other question here: should we also be removing directories if they are empty after removing low-value files?  Personally, I think we should. when I run `qri remove --all --force` I'm trying to blow the directory away.

We could save that for another PR, but if we can get it knocked out now, I'd be all for it

Reply:
Got covered as part of the update :)